alcs-frontend/src/app/features/board/board.component.ts

```diff
@@ -347,11 +348,13 @@ export class BoardComponent implements OnInit, OnDestroy {
   }
 
   private mapReconsiderationDtoToCard(recon: ApplicationReconsiderationDto): CardData {
+    const mappedType = this.applicationTypes.find((type) => type.code === recon.application.type.code);
```

Comment:
If you already have the full type under recon.application.type, why do you need to find it?